README.md

```diff
@@ -33,3 +33,109 @@ Prepare the package:
+               --publish 9300:9300 \
+               --rm \
+               --user $UID \
+               --volume /tmp/es:/usr/share/elasticsearch/data \
+               elasticsearch:7.10.1
+    ```  
+    Start the elastic search engine and make sure that any of the data created is owned by yourself so that you can clean it up. If you are not using the /tmp/es directory created above, then make sure to change the --volume argument as needed so that elastic search has access to it. Also change the --user arguement such that elastic search has full privledges to read, write, and modify the data.
+
+### Install test data 
+
+This section will use the small test data set contained within the docker image. The steps used to do this can be used as a template to install your own more complicated datasets as well.
+
+First, install the registry schema for elastic search. Things to note, the URL host name is the --name argument value used when starting elastic search.
+
+```
+docker run --network pds \
+           --rm \
+           --user $UID \
+           pds_registry_app:$(git rev-parse HEAD) \
+              registry-manager create-registry -es http://es:9200 \
+                                               -schema /var/local/registry/elastic/registry.json 
+```
+
+Second we need to harvest the test data. It is currently setup in /var/local/harvest/archive. If one wanted to use their own archive of data, they would simply have to add `--volume /path/to/archive:/var/local/harvest/archive` to the docker run portion of this command. Because we cannot easily run two commands in one docker run invokation, it is necessary to use an external directory for /var/local/havest/output. The ephemeral nature of a container erases what we would write into /var/local/harvest/output between the two docker run invokations.
+```
+docker run --network pds \
+           --rm \
+           --user $UID \
+           --volume /tmp/output:/var/local/harvest/output \
+           pds_registry_app:$(git rev-parse HEAD) harvest \
+              -c /var/local/harvest/conf/examples/registry.xml \
+              -o /var/local/harvest/output
+```
+
+Third, and last, we need to now ingest the havested data from above into elastic search. Things to note: One, the URL host name is the --name argument value used when starting elastic search. Two, the output volume must be the same mapping as in the previous step.
+```
+docker run --network pds \
+           --rm \
+           --user $UID \
+           --volume /tmp/output:/var/local/harvest/output \
+           pds_registry_app:$(git rev-parse HEAD) \
+           registry-manager load-data \
+              -es http://es:9200 \
+              -file /var/local/harvest/output
+```
+
+### Verify it worked
+
+The verification script is written in Python 3.x with requests installed as well only because that is what the person doing the work knows the best. A simple curl then lots of scrolling and eyeballing can do the same thing.
```

Comment:
An easy alternative to running the python script can be to request elasticsearch in a browser or with curl (http://localhost:19200/registry/_search) but the script is good since it checks that all the data are ingested.